### PR TITLE
issue#4_solved

### PR DESCRIPTION
**Pull request by zjp292**
added download button to events page

### CODE REVIEW

File: weapon-system-support-software/mainwindow.ui
Comment (zjp292) [inline weapon-system-support-software/mainwindow.ui:1]: I am not sure why this changed so much, maybe issue

Comment (bradd07) [reply weapon-system-support-software/mainwindow.ui:1]: bad programming

Comment (zjp292) [reply weapon-system-support-software/mainwindow.ui:1]: :(

**Comment by Dylan-Motz** [comment]
Looks good to me

**Pull request merged**